src/controllers/server.js

```diff
@@ -79,10 +79,12 @@ class Server extends EventEmitter {
             if (err && err.statusCode === 404) { // no such container
                 this.log.info('Container was not found. Attempting to recreate it.');
                 next(); // Continue normal initialization
-                this.rebuild(() => {
-                    this.log.fatal('Could not recreate container.');
+                this.rebuild((rebuildErr) => {
```

Comment:
This fails eslint checking, `rebuildErr` should not be surrounded by parentheses.